dali/test/python/test_operator_rotate.py

```diff
@@ -48,6 +52,45 @@ def get_output_size(angle, input_size):
         out_h += 1
     return (out_h, out_w)
 
+
+def get_3d_lin_rotation(angle, axis):
+  # mirrors transform.h:rotation3D
+  if not angle:
+    return np.eye((3, 3), dtype=np.float32)
+  axis_norm = np.linalg.norm(axis)
+  axis = [dim / axis_norm for dim in axis]
+  u, v, w = axis
+  cosa = math.cos(angle)
+  sina = math.sin(angle)
+  return np.array([
+    [u*u + (v*v+w*w)*cosa, u*v*(1-cosa) - w*sina, u*w*(1-cosa) + v*sina],
+    [u*v*(1-cosa) + w*sina, v*v + (u*u+w*w)*cosa, v*w*(1-cosa) - u*sina],
+    [u*w*(1-cosa) - v*sina, v*w*(1-cosa) + u*sina, w*w + (u*u+v*v)*cosa],
+  ], dtype=np.float32)
+
+
+def get_3d_output_size(angle, axis, input_size, parity_correction=False):
+  rotation = np.abs(get_3d_lin_rotation(angle, axis))
```

Comment:
Nitpick: Some other name would be nice - this is not really a rotation matrix...

Reply:
done